### ISSUE TITLE
Connection started to drop in a minute after I replace my smartphone.

### ISSUE THREAD

**Issue by lamrongol** [musicbeeremote/mbrc#240]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

First, this problem probably only happens in certain cases and we'll be not able to figure out what caused it. I'll leave it as a record just in case.

Recently I replaced my smartphone from Galaxy S9 to Galaxy S20 and re-installed MusicBee remote. Then, I noticed MusicBee remote on Galaxy S20 connection drops in a minute even if I stay in home all the time. 
I confirmed MusicBee remote on Galaxy S9 connection doesn't drop as long as I'm at home.

Maybe, the cause is that OS of Galasy S9 is Android 10 but Galaxy S20 is Android S20 

### Environment

```markdown
- App Version: v1.5.1
Android version and device is as above
```

**Comment by lamrongol** [comment]
Thank you for reply. 
>Also maybe you could enable logs on the app on the S20 and leave it until this happens, and then send me the logs?
I found Debug Logging option but can't find logs. Should I send from feedback menu?

**Comment by kelsos** [comment]
> @kelsos Thank you for reply.
> 
> > Also maybe you could enable logs on the app on the S20 and leave it until this happens, and then send me the logs?
> 
> I found Debug Logging option but can't find logs. Should I send from feedback menu?

Yes, it will create a zip archive and it will attach it along with the mail from the feedback menu.

I managed to find some time to check stuff today. I am running a test on a Pixel 6 and on a Galaxy Tab S6 Lite to see how things go.

If you manage to send me the logs this will also help.

**Comment by lamrongol** [comment]
@kelsos I send the logs.

**Comment by kelsos** [comment]
@lamrongol sorry for taking too long to reply, I am looking at the logs. In a lot of cases, it seems that after a while the application stops receiving pings and disconnects. I will see if I can figure out something, but in the meanwhile could you try the instructions on the site [https://dontkillmyapp.com/samsung](https://dontkillmyapp.com/samsung) to see if this helps?

Apparently, Samsung is now doing an even more aggressive power management that kills even foreground applications which might explain what you see.

I will check if I can figure anything else but also let me know if changing the settings helps at all.

**Comment by lamrongol** [comment]
@kelsos I set app setting "(Battery) Don’t optimize" and connection no longer drop for at least three days. Thank you very much.

**Comment by kelsos** [comment]
Great, good to know that this works for you. I need to add it to an FAQ somewhere now.
